prog/postgres/postgres_resource_nexus.rb

```diff
@@ -40,13 +43,13 @@ def self.assemble(project_id:, location:, name:, target_vm_size:, target_storage
             parent.timeline.latest_restore_time && restore_target <= parent.timeline.latest_restore_time
           fail Validation::ValidationFailed.new({restore_target: "Restore target must be between #{earliest_restore_time} and #{parent.timeline.latest_restore_time}"})
         end
-        [parent.superuser_password, parent.timeline.id, "fetch"]
+        [parent.superuser_password, parent.timeline.id, "fetch", parent.version]
```

Comment:
Should we trigger an error when the parent and version are provided at the same time? Otherwise, the given version might be silently ignored, potentially leading to operator mistakes.